occ_manager.cpp

```diff
@@ -544,7 +516,12 @@ void Manager::statusCallBack(instanceID instance, bool status)
                 "COUNT", activeCount, "INST", instance, "STATUS", status);
         }
         // Clear OCC sensors
-        setSensorValueToNaN(instance);
+        for (auto& obj : statusObjects)
+        {
+            // int instance = obj->getOccInstanceID();
```

Comment:
This loop is setting all OCCs sensors to NAN vs just the single instance that went away. I assume you need to add the check that was in the function you removed.